src/VisualStudio/Core/Impl/CodeModel/FileCodeModel.cs

```diff
@@ -414,19 +420,22 @@ internal SyntaxNode GetSyntaxRoot()
             return GetDocument().GetSyntaxRootSynchronously(CancellationToken.None);
         }
 
+        internal DocumentOptionSet GetDocumentOptions(CancellationToken cancellationToken)
+            => State.ThreadingContext.JoinableTaskFactory.Run(() => GetDocument().GetOptionsAsync(cancellationToken));
```

Comment:
❔ Why are we adding this?

Reply:
Removed after reviewing comment on hiding async work